domains/romeo.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Strvm",
+        "email": "[email protected]"
```

you can remove the email if you dont want to share it
```suggestion
        "username": "Strvm"
```